src/Http/Response/Created.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+/**
+ * File contains the PSR7 ResponseInterface implementation for
+ * Created responses.
+ *
+ * PHP Version 7
+ *
+ * @category PSR15
+ * @package  Http
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ *
+ * @see https://www.php-fig.org/psr/psr-7/
+ * @see https://www.php-fig.org/psr/psr-15/
+ */
+namespace LORIS\Http\Response;
+
+use \LORIS\Http\Response\JsonResponse;
+
+/**
+ * A LORIS Http Response is an implementation of the PSR15 ResponseInterface
+ * to use in LORIS specifically for the 201 Created response.
+ *
+ * @category PSR15
+ * @package  Http
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Created extends JsonResponse
+{
+    /**
+     * Create a Json response specific to 201 Created
+     *
+     * @param array  $body     The response content
+     * @param string $location The created resource location
+     *
+     * @return void
+     */
+    public function __construct(array $body = [], string $location = '/')
```

Comment:
Please remove the default value of `$location`.
The location header must be used when the url of the new resource is different than the url used to create the resource. (e.g.: POST /candidates => 201 Created 'location: /candidates/$candid' )

Reply:
@johnsaigle I see that you need to put the location parameter first because I ask that the default value be removed but it creates a counter-intuitive usage (imo).
Why not simply stick with the same constructor than the OK with `array $body = [], array $headers = []` byut add a check to make sure that if the body is empty, that there is a location header?

Reply:
I don't feel strongly about this one way or another, but I don't really find "location, body" counter-intuitive..

Reply:
@driusan It's not counter-intuitive in itself but all the other response classes have `body` first.

@xlecours I'll update the code with your suggestion.